lib/puppet/version.rb

```diff
@@ -8,7 +8,7 @@
 # Raketasks and such to set the version based on the output of `git describe`
 
 module Puppet
-  PUPPETVERSION = '8.22.0'
+  PUPPETVERSION = '8.22.1'
```

Comment:
Can we commit this change directly in the main branch (or at least, not have it in the middle of this feature branch, because once merged it will be weird)?

Reply:
Can do!